### PR TITLE
Issue 7 done

### PR DESCRIPTION
**Pull request by MakarSerhii**
Created User model, request to user for giving the phone number, taking email from user with "re" validation, stored chat_id, number, and email to the db, created identification by number

### CODE REVIEW

**Comment by MakarSerhii** [comment]
done

File: scripts/bot.py [hunk 1]
```diff
@@ -74,9 +125,11 @@ def main():
     dp.add_handler(CommandHandler("report", report, run_async=True))
     dp.add_handler(CommandHandler("save_reports", save_reports))
     dp.add_handler(MessageHandler(Filters.text, code))
+    dp.add_handler(MessageHandler(Filters.contact, get_number))
+    dp.add_error_handler(error)
```
Comment (SergeyKutsko) [inline scripts/bot.py:129]: @MakarSerhii please remove that old handler

Comment (MakarSerhii) [reply scripts/bot.py:129]: I need this 1 handler 
dp.add_handler(MessageHandler(Filters.contact, get_number))

File: scripts/bot.py [hunk 2]
```diff
@@ -122,14 +117,10 @@ def main():
     updater = Updater(os.environ['TELEGRAM_TOKEN'], use_context=True)
     dp = updater.dispatcher
     dp.add_handler(CommandHandler("start", start))
-    dp.add_handler(CommandHandler("report", report, run_async=True))
-    dp.add_handler(CommandHandler("save_reports", save_reports))
-    dp.add_handler(MessageHandler(Filters.text, code))
     dp.add_handler(MessageHandler(Filters.contact, get_number))
-    dp.add_error_handler(error)
```
Comment (SergeyKutsko) [inline scripts/bot.py]: @MakarSerhii sorry was not very descriptive, but you should only delete `dp.add_error_handler(error)` and back all other handlers

Comment (MakarSerhii) [reply scripts/bot.py]: Done

**Comment by SergeyKutsko** [comment]
@MakarSerhii looks good, 1 time to fix left

**Review by SergeyKutsko — APPROVED** [review]
@MakarSerhii please fix last comment

**Comment by SergeyKutsko** [comment]
@MakarSerhii please back deleted file [docker-compose_example.yml](https://github.com/SergeyKutsko/universal_auto/pull/29/files#diff-bbeedf7ede917dc7f942e2566928b6e45aec447be890af6573885a8b495addb0)

**Comment by MakarSerhii** [comment]
fixed

**Comment by SergeyKutsko** [comment]
@MakarSerhii please add basic tests

**Comment by SergeyKutsko** [comment]
@MakarSerhii please add basic tests to confirm your code works as expected. i give you 1 extra day for that, thanks

**Comment by MakarSerhii** [comment]
i need to do the tests for User model?

**Comment by MakarSerhii** [comment]
Basic tests added

**Comment by SergeyKutsko** [comment]
@MakarSerhii here is your Interview link: https://calendly.com/universal_auto_internship/30min

**Pull request merged**